### PR TITLE
Cyclone5 webkit

### PR DESCRIPTION
**Pull request by fahadusman**

### CODE REVIEW

**Comment by kergoth** [comment]
At least some of this appears to be general fixes that should go upstream, no? Shouldn't this be split up into both meta-mel and meta-mentor-staging?

**Comment by fahadusman** [comment]
@kergoth yes, the packaging on debug files need to be pushed upstream

**Comment by fahadusman** [comment]
@Noor-Ahsan, @kergoth I have updated the pull request and moved that change to meta-mentor-staging.

File: meta-mel/meta-altera/recipes-sato/webkit/webkitgtk_2.10.7.bb [hunk 1]
```diff
@@ -76,3 +76,5 @@ FILES_${PN} += "${libdir}/webkit2gtk-4.0/injected-bundle/libwebkit2gtkinjectedbu
 
 # http://errors.yoctoproject.org/Errors/Details/20370/
 ARM_INSTRUCTION_SET = "arm"
+
+DEBUG_FLAGS_remove = "-g"
```
Comment (kergoth) [inline meta-mel/meta-altera/recipes-sato/webkit/webkitgtk_2.10.7.bb]: This failure doesn't occur with non-mel builds?

Comment (fahadusman) [inline meta-mel/meta-altera/recipes-sato/webkit/webkitgtk_2.10.7.bb]: @kergoth this only occurs when we build webkitgtk with Codebench, and it builds fine with internal toolchain.

**Pull request merged**